irohad/validation/impl/stateful_validator_impl.cpp

```diff
@@ -143,81 +148,55 @@ namespace iroha {
      * @param log to write errors to console
      * @return vector of proto transactions, which passed stateful validation
      */
```

Comment:
Update the doc with new parameters, please

Reply:
Also, as I understand, returned value is not a vector anymore

Reply:
https://github.com/hyperledger/iroha/blob/6e49d70c2e17f20a612bce1b334a91d83f5be4b3/irohad/validation/impl/stateful_validator_impl.cpp#L149-L150